test/stmt/if_unexpected_else.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
No, I think we'd rather get all the way through Sema, to make sure we don't build an invalid AST.